.babelrc

```diff
@@ -1,20 +1,23 @@
 {
-  "stage": 0,
+  "presets": ["es2015", "react", "stage-0"],
   "env": {
     "browser": {
       "plugins": ["react-transform"],
```

Comment:
Yes thank's @justingreenberg, someone published a preset for this: https://github.com/danmartinez101/babel-preset-react-hmre

I'm waiting for babel-plugin-react-transform tu be updated first to make the choice :+1: